packages/web-react/src/types/shared/tokens.ts

```diff
@@ -1,4 +1,6 @@
 import { breakpoints, space } from '@lmc-eu/spirit-design-tokens';
 
+export const BREAKPOINT_MOBILE = 'mobile';
```

Comment:
Is there a type for this…?

```suggestion
export const BREAKPOINT_MOBILE = breakpoints.MOBILE;
```

Reply:
Nope, this would return the value 0, i need the name...